composer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "ciprianmatis/masterclass-repo",
+    "description": "Masterclass",
+    "require-dev": {
+        "phpunit/phpunit": "~4.7@dev"
```

Comment:
You'll want to make sure and track a stable version of packages. 4.5 is current.